shell/platform/common/client_wrapper/core_implementations.cc

```diff
@@ -164,6 +166,51 @@ void ReplyManager::SendResponseData(const std::vector<uint8_t>* data) {
 
 }  // namespace internal
 
+// ========== method_channel.h ==========
+
+namespace internal {
+
+static constexpr char kControlChannelName[] = "dev.flutter/channel-buffers";
+static constexpr char kResizeMethod[] = "resize";
+static constexpr char kOverflowMethod[] = "overflow";
```

Comment:
These three lines should be in an anonymous namespace (and thus don't need `static` to have internal linkage), rather than the `internal` namespace, since they don't need to be referenced outside of this file.